pkg/apis/storage/types.go

```diff
@@ -283,8 +283,8 @@ type CSIDriverSpec struct {
 	// Defines if the underlying volume supports changing ownership and
 	// permission of the volume before being mounted.
 	// Refer to the specific FSGroupPolicy values for additional details.
-	// This field is beta, and is only honored by servers
-	// that enable the CSIVolumeFSGroupPolicy feature gate.
+	// This field is only honored by servers that enable the
```

Comment:
I think we can remove this line altogether as the feature is locked to on and can't be disabled.

Reply:
Also update the versioned (v1, v1beta, etc) types.go as well.